### PR TITLE
Fix assertion

### PR DESCRIPTION
**Pull request by tylerbenson**
Statement asserts don't work inside if blocks.

### CODE REVIEW

**Review by richardstartin — APPROVED** [review]

File: dd-java-agent/instrumentation/jetty-7.0/src/test/groovy/Jetty70Test.groovy [hunk 1]
```diff
@@ -111,7 +111,9 @@ class Jetty70Test extends HttpServerTest<Server> {
           response.sendRedirect(endpoint.body)
           break
         case ERROR:
-          response.sendError(endpoint.status, endpoint.body)
+        // response.sendError(endpoint.status, endpoint.body)
+          response.status = endpoint.status
+          response.writer.print(endpoint.body)
```
Comment (bantonsson) [inline dd-java-agent/instrumentation/jetty-7.0/src/test/groovy/Jetty70Test.groovy:116]: How is this change related to the `assert` change, and should the comment be left in there?

Comment (tylerbenson) [reply dd-java-agent/instrumentation/jetty-7.0/src/test/groovy/Jetty70Test.groovy:116]: This was necessary because the change in the assert caused the body to actually be validated which this test was not sending properly. I left the comment there more as a reference. Would you prefer if I add a comment explaining this or remove the existing comment?

Comment (bantonsson) [reply dd-java-agent/instrumentation/jetty-7.0/src/test/groovy/Jetty70Test.groovy:116]: A quick comment explaining that above the `// response.sendError...` would be great.

**Review by bantonsson — APPROVED** [review]
